File: web/src/main/vite-project/src/kostratypes/kostraFormTypeVm.ts
import Nullable from "./nullable"

interface KostraFormTypeVm {
    readonly id: NonNullable<string>
    readonly tittel: NonNullable<string>
    readonly labelOrgnr: Nullable<string>
    readonly labelOrgnrVirksomhetene: Nullable<string>
}

export default KostraFormTypeVm

File: application-common/src/main/kotlin/no/ssb/kostra/common/viewmodel/KostraFormTypeVm.kt
package no.ssb.kostra.common.viewmodel

import io.micronaut.core.annotation.Introspected

@Introspected
data class KostraFormTypeVm(
    val id: String,
    val tittel: String,
    val labelOrgnr: String?,
    val labelOrgnrVirksomhetene: String?
)
